workflows/proteomics/clinicalmp/clinicalmp-database-generation/iwc-clinicalmp-database-generation.ga

```diff
@@ -0,0 +1,348 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Generating a customized database for clinical metaproteomics",
```

Comment:
Can you say here what the workflow does ?
Something like what you wrote in the readme ?
> The first step in this workflow is the Database Generation process. The Galaxy-P team has created a workflow that compiles a large database by downloading protein sequences of known disease-causing microorganisms. From this extensive database, a compact, relevant database is then created using the Metanovo tool.